ui/pages/bridge/index.tsx

```diff
@@ -93,41 +96,51 @@ const CrossChainSwap = () => {
   return (
     <div className="bridge">
       <div className="bridge__container">
-        <Header
-          className="bridge__header"
-          startAccessory={
-            <ButtonIcon
-              iconName={IconName.ArrowLeft}
-              size={ButtonIconSize.Sm}
-              ariaLabel={t('back')}
-              onClick={redirectToDefaultRoute}
-            />
-          }
-          endAccessory={
-            <ButtonIcon
-              iconName={IconName.Setting}
-              size={ButtonIconSize.Sm}
-              ariaLabel={t('settings')}
-            />
-          }
-        >
-          {t('bridge')}
-        </Header>
-        <Content className="bridge__content">
-          <Switch>
-            <FeatureToggledRoute
-              redirectRoute={SWAPS_MAINTENANCE_ROUTE}
-              flag={isBridgeEnabled}
-              path={CROSS_CHAIN_SWAP_ROUTE + PREPARE_SWAP_ROUTE}
-              render={() => {
-                return <PrepareBridgePage />;
-              }}
-            />
-          </Switch>
-        </Content>
-        <Footer>
-          <BridgeCTAButton />
-        </Footer>
+        <Switch>
+          <FeatureToggledRoute
+            redirectRoute={SWAPS_MAINTENANCE_ROUTE}
+            flag={isBridgeEnabled}
+            path={CROSS_CHAIN_SWAP_ROUTE + PREPARE_SWAP_ROUTE}
+          >
+            <>
+              <Header
+                className="bridge__header"
+                startAccessory={
+                  <ButtonIcon
+                    iconName={IconName.ArrowLeft}
+                    size={ButtonIconSize.Sm}
+                    ariaLabel={t('back')}
+                    onClick={redirectToDefaultRoute}
+                  />
+                }
+                endAccessory={
+                  <ButtonIcon
+                    iconName={IconName.Setting}
+                    size={ButtonIconSize.Sm}
+                    ariaLabel={t('settings')}
+                  />
+                }
+              >
+                {t('bridge')}
+              </Header>
+              <Content className="bridge__content">
+                <PrepareBridgePage />
+              </Content>
+              <Footer>
+                <BridgeCTAButton />
+              </Footer>
+            </>
+          </FeatureToggledRoute>
+
+          <Route path={CROSS_CHAIN_SWAP_ROUTE + AWAITING_SIGNATURES_ROUTE}>
+            <Content className="bridge__content">
```

Comment:
You can remove the className here since there's no scss style for it

Reply:
Done